### PR TITLE
Fix typos for shortnames

### PR DESCRIPTION
**Pull request by squirrelo**
Part of labadmin pulldown fix [#31](https://github.com/biocore/labadmin/issues/31)

### CODE REVIEW

**Comment by squirrelo** [comment]
@mortonjt can this get a quick review?

File: amgut/db/patches/0030.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+--November 7, 2015
+--Fix typos in column shortnames
+UPDATE ag.survey_question SET question_shortname = 'FROZEN_DESSERT_FREQUENCY' WHERE survey_question_id = 66;
+UPDATE ag.survey_question set question_shortname = 'OTHER_CONDITIONS_LIST' WHERE survey_question_id = 106;
```
Comment (mortonjt) [inline amgut/db/patches/0030.sql]: Can we have consistent casing?

Comment (squirrelo) [inline amgut/db/patches/0030.sql]: Yup, fixed.

**Comment by mortonjt** [comment]
Looks good!  Since this is such a small PR, would be advantageous to include the rest of the fixes in https://github.com/biocore/labadmin/issues/31 ?

**Comment by squirrelo** [comment]
There's already a PR in labadmin addressing the rest of them. This one just needed database changes.

**Comment by mortonjt** [comment]
OK.  So is this ready to merge then?

**Comment by squirrelo** [comment]
yup

**Pull request merged**